static/js/publisher/release/components/releasesConfirmDetails.js

```diff
@@ -0,0 +1,306 @@
+import React, { Fragment, useState } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+
+import { updateProgressiveReleasePercentage } from "../actions/pendingReleases";
+import { isProgressiveReleaseEnabled } from "../selectors";
+
+import { ProgressiveBar, InteractiveProgressiveBar } from "./progressiveBar";
+
+const progressiveTypes = {
+  RELEASE: "release",
+  UPDATE: "update",
+  CANCELLATION: "cancel"
+};
+
+const ReleaseRow = ({ type, revisionInfo, channel, progress, notes }) => (
+  <div className="p-releases-confirm-details__row">
```

Comment:
If it's a component of its own, probably good candidate for 'top level' pattern naming: `p-release-row` (?)
This would simplify deeply nested class names.